lld/MachO/SyntheticSections.h

```diff
@@ -315,6 +315,25 @@ class StubHelperSection final : public SyntheticSection {
   Defined *dyldPrivate = nullptr;
 };
 
+class ObjCSelRefsSection final : public SyntheticSection {
+public:
+  ObjCSelRefsSection();
+  void initialize();
+
+  // This SyntheticSection does not do its own writing, but instead uses the
+  // creates SyntheticInputSection's and inserts them into inputSections
+  uint64_t getSize() const override { return 0; }
+  bool isNeeded() const override { return false; }
+  void writeTo(uint8_t *buf) const override {}
+
+  ConcatInputSection *getSelRefForMethName(StringRef methName);
+  ConcatInputSection *makeSelRef(StringRef methName);
```

Comment:
It seems the return value is not used. Perhaps `void createSelRef` or `ConcatInputSection *getOrCreate`? It would be great if you can comment on these function declarations here.

Reply:
Yes, not used right now, I am planning to use it in the future though.

Reply:
Okay. To make these more consistent, you can also shorten `getSelRefForMethName` into `getSelRef`.